app/shared_features/set_card/SetForm.js

```diff
@@ -70,13 +70,15 @@ class SetForm extends Component {
             weight: weight,
             didChangeWeight: true
         });
+        this.props.saveSet(this.props.setID, this.state.exercise, this.state.weight, this.state.metric, this.state.rpe);
     }
 
     _onChangeRPE(rpe) {
         this.setState({
             rpe: rpe,
             didChangeRPE: true
         });
+        this.props.saveSet(this.props.setID, this.state.exercise, this.state.weight, this.state.metric, this.state.rpe);
     }
 
     // SAVING
```

Comment:
these run way too often, want to do it on losing focus or something not on EACH digit typed